requirements/static/docs.in

```diff
@@ -0,0 +1 @@
+sphinx
```

Comment:
Should we require sphinx greater than or equal to 1.7.1 since we've made modifications that require that now?

Reply:
We may even need a newer version such as 1.7.2 than that to avoid warnings.
1.8.5 on python2 and 2.0.1 on python3 would be ideal.

Reply:
I supposed that should be done in a separate pr though, I just saw this and thought about it.

Reply:
We should definitely bring in the work you've done, but yeah, lets do that on a separate PR.